app/services/set_participant_categories.rb

```diff
@@ -27,41 +27,49 @@ def set_participant_categories
   end
 
   def fip_participant_categories_feature_flag_active
-    ParticipantCategories.new(eligible_participants, fip_flag_active_ineligible_participants, contacted_for_info_participants, details_being_checked_participants)
+    ParticipantCategories.new(eligible_participants, fip_flag_active_ineligible_participants, fip_flag_active_withdrawn_participants, contacted_for_info_participants, details_being_checked_participants)
   end
 
   def fip_participant_categories_feature_flag_inactive
-    ParticipantCategories.new([], [], contacted_for_info_participants, fip_flag_inactive_details_being_checked_participants)
+    ParticipantCategories.new([], [], withdrawn_participants, contacted_for_info_participants, fip_flag_inactive_details_being_checked_participants)
   end
 
   def cip_participant_categories
-    ParticipantCategories.new(cip_eligible_participants, [], contacted_for_info_participants, [])
+    ParticipantCategories.new(cip_eligible_participants, [], withdrawn_participants, contacted_for_info_participants, [])
   end
 
   def active_participant_profiles
     ParticipantProfilePolicy::Scope.new(user, school_cohort.active_ecf_participant_profiles).resolve
   end
 
   def ineligible_participants
-    active_participant_profiles.ineligible_status.includes(:user).order("users.full_name")
+    active_participant_profiles.ineligible_status.includes(:user).order("users.full_name").where.not(training_status: :withdrawn)
   end
 
   def eligible_participants
-    active_participant_profiles.eligible_status.includes(:user).order("users.full_name")
+    active_participant_profiles.eligible_status.includes(:user).order("users.full_name").where.not(training_status: :withdrawn)
+  end
+
+  def withdrawn_participants
+    active_participant_profiles.training_status_withdrawn.includes(:user).order("users.full_name")
   end
 
   def contacted_for_info_participants
-    active_participant_profiles.contacted_for_info.includes(:user).order("users.full_name")
+    active_participant_profiles.contacted_for_info.includes(:user).order("users.full_name").where.not(training_status: :withdrawn)
   end
 
   def details_being_checked_participants
-    active_participant_profiles.details_being_checked.includes(:user).order("users.full_name")
+    active_participant_profiles.details_being_checked.includes(:user).order("users.full_name").where.not(training_status: :withdrawn)
   end
 
   def fip_flag_active_ineligible_participants
     ineligible_participants - eligible_participants
```

Comment:
Where does a withdrawn ERO mentor appear?

Reply:
They should appear in the withdrawn section. I've changed the training status on "Previous Participation ERO" user test data and they show up there.